rqt_gps_rtk_plugin/CMakeLists.txt

```diff
@@ -1,119 +1,146 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(rqt_gps_rtk_plugin)
 
-# Set compiler flags
-set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
+## Use C++11
+## By adding -Wall and -Werror, the compiler does not ignore warnings anymore,
+## enforcing cleaner code.
+add_definitions(-std=c++11 -Wall -Werror)
 
+# Load catkin and all dependencies required for this package
 find_package(catkin REQUIRED COMPONENTS
-   any_worker
-   rqt_gui
-   rqt_gui_cpp
-   piksi_rtk_msgs
-   sensor_msgs
+  rqt_gui
+  rqt_gui_cpp
+  any_worker
```

Comment:
since we depend on any_worker, can you please put clone instruction in the read me of the gui to clone the repo "any_common"?